### DISCUSSION TITLE
话说以后这个项目大了之后会不会有一些低龄低素质的人啊？林子大了什么人都有，b站的社区环境已经搞得乌烟瘴气的了，有点怕这里也会沦陷

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#2325] [其他]
<img width="1440" alt="截屏2024-07-05 21 56 22" src="https://github.com/user-attachments/assets/d2fe5f4e-fb48-4b43-a688-603e167a621e">
各位绳匠来讨论一下

**Comment by KELERSAMA** [comment]
进得来是前提吧

**Comment by 369852147abc** [comment]
不会，光是注册账号都有门槛

**Reply by jane-10086-jpg** [reply]
确实，而且很多人也不会随便来这里发表什么言论，这里发言可比知乎什么难多了

**Comment by Huanshenxuanlong** [comment]
不吹不黑，进来的门槛还是有的

**Comment by WRYYY-S** [comment]
别的不说，可能xxs连等待github加载的耐心都没有，众所周知github在国内访问性比较差，而且不是用手机一个点击下载和手机号一键注册就可以搞定的

更何况要在github上开团就不是一条5毛了

**Reply by Mai-ther** [reply]
我不用点手段，连帖子都点不开……

**Comment by xiaochenyu24** [comment]
xxs连github怎么进都不知道

**Comment by yanlei2009** [comment]
进来了要搞事也要会一点英文的
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Phaethon-0019** [comment]
github这个账号注册难度就比普通账号要高不少，跟何况xxs也不知道怎么挂脚本加速，只能说很难，但完全没有不太可能

**Comment by FNN66** [comment]
因该不会，毕竟进入绳网的门槛也不小，估计早就劝退了一些人

**Comment by soyobat** [comment]
来这的太多是程序员吧

**Reply by nninng** [reply]
并不是，我只是一个高中生，搜教程摸爬滚打2小时进来了( ͡° ͜ʖ ͡°)✧

**Reply by LITUKA111** [reply]
教程来的(〜￣▽￣)〜

**Reply by tomppap** [reply]
大佬们，怎么创建一个讨论频道。我还真不会求指导

**Reply by skycat2216** [reply]
新的討論主題？發文模板消失了，除非作者起床否則基本沒辦法了

**Comment by 6078HITHEDAY** [comment]
# 大不了，删库跑路

**Reply by nostalgiatan** [reply]
我勒个删库跑路啊

**Reply by RyanLosTRxx** [reply]
原来是这样解决的吗（

**Reply by shouyeren7** [reply]
我嘞个删库跑路啊

**Reply by Chihaya-Yuka** [reply]
这个解决方法真是摇摇又滚滚啊。

**Reply by blacktree-eme** [reply]
原来是用这种方式解决的吗？

**Reply by LilangAlan** [reply]
我嘞个删库跑路啊

**Reply by bangbu01** [reply]
你难道是天才？

**Reply by 660W** [reply]
我嘞个删库跑路啊
**你难道是天才？**

**Reply by H02681** [reply]
罄竹难书了

**Reply by bbdqz** [reply]
添柴!

**Reply by 11-billy** [reply]
6666

**Reply by zei-21** [reply]
我：好一个删库跑路

**Comment by xiaomwm** [comment]
爱门♫♫♫

**Comment by gawrgura-qing** [comment]
我是小学生doge

**Reply by bbdqz** [reply]
小初生了解一下(
![68747470733a2f2f75706c6f61642d6262732e6d69796f757368652e636f6d2f75706c6f61642f323032332f30332f30392f33313733626333653639666366656265636664613734313334306339616432325f363834313235303331303735363237353637382e706e67](https://github.com/user-attachments/assets/dab80040-d090-4d9d-8a9d-33656509b558)

**Comment by Sqrootof** [comment]
那不是进了程序员老巢么，这不得狠狠按住输出，**你疑似对米哈游不忠了！**<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by JobNumber-9527** [comment]
油猴+插件+加速器+注册账号错一步都是垃圾软件满天飞

**Comment by wtnl551** [comment]
这应该不太可能，进入门槛有点高

**Comment by DrAshewell** [comment]
注册账号对于他们来说就像打11级尼尼微一样艰难，开怀大笑

**Reply by ysd1911** [reply]
哒咩，11侵蚀尼尼微还是好打的，也就凹了一个上午，但是绳网不看是真不知道咋进来

**Comment by Kioeco** [comment]
比较难吧，毕竟它们没脑子

**Reply by nostalgiatan** [reply]
有限的门槛成功挡住了大部分人，不过还是要小心小部分人进来浑水摸鱼的。

**Reply by homteam** [reply]
> 比较难吧，毕竟它们没脑子

还好我刚毕业不然就没脑子了😅

**Comment by kinght-glitch** [comment]
哇，这也能遇到爱门啊。

**Comment by RyanLosTRxx** [comment]
github就能筛掉95%的人了

**Reply by homteam** [reply]
> github就能筛掉95%的人了

诶🤓👆，如果软件做出来了放到steam里面是不是就可以筛选掉87%+95%=182%的中国人了！(bushi

**Reply by ysd1911** [reply]
难道你真的是天才？

**Reply by blacktree-eme** [reply]
> > github就能筛掉95%的人了
> 
> 诶🤓👆，如果软件做出来了放到steam里面是不是就可以筛选掉87%+95%=182%的中国人了！(bushi

数学老师：我是这么教你的？

**Reply by homteam** [reply]
> 数学老师：我是这么教你的？
有道理🧐
对了🤓👆我可以用乘法😋
87%*95%=8265%👍
我太聪明啦💀

**Reply by fool-2422** [reply]
> > 数学老师：我是这么教你的？
> > 有道理🧐
> > 对了🤓👆我可以用乘法😋
> > 87%*95%=8265%👍
> > 我太聪明啦💀

天才😋

**Comment by 00zzxx** [comment]
虽然说门栏不高，但是还是有一些的，大不了，大不了就删库跑路嘛<img src="https://bbs-static.miyoushe.com/static/2023/07/12/61bc6bb036f05c6deb3b6e8d21699234_2780715002462542354.png" alt="帕姆 哭哭" width="75" >

**Comment by Yoimiya-M** [comment]
可能会有个别的，但是大规模出现完全不可能。
再说这里有一个举报处，个别的你可能没看见就被举报没了。

**Comment by UEG-LmR** [comment]
这里可是Github先不说xxs知不知道有着个东西光挂加速，注册，下油猴就拦住不少人（机器人）
再者Github的国内访问不稳定，遭不住瞬时大批量访问。（本人是个中学生如果说的不对不要当真）

**Reply by cyq20091029** [reply]
鄙人也是个准初三，感觉门槛高是件好事

**Comment by JIIIMXu** [comment]
站主好像有管理（？），反正也有举报按钮，大家伙人多力量大嘛

**Comment by HKLHaoBin** [comment]
我感觉不会，发评论至少需要：

1. 有电脑
2. 会科学上网
3. 会使用浏览器插件

![Clip_2024-08-18_00-35-36](https://github.com/user-attachments/assets/c8ee9778-4367-4bc3-bee5-c7ac76f5d1cd)

**Reply by shxtxdy** [reply]
手机不就可以吗，其实总体下来感觉门槛也没多高，网站要是火的话，肯定会有一些渗透进来，并且不会少，当然前提是要很火，现在年龄越小，接触的东西相比于我们都更早，我小时候还在地里玩泥巴，再看看他们现在的，网站太火的话，被渗透是迟早的事😌

**Reply by HKLHaoBin** [reply]
手机不行，没办法弄篡改猴，但是可以直接进 github ，前提是能看懂。

**Reply by shxtxdy** [reply]
现在手机都自带翻译器，而且还有的浏览器也自带，语言障碍在现在都不成问题了，更何况现在网上还有教程什么的，只要火了肯定有大批量的xxs进来，删站跑路是个好战术🌚

**Reply by HitoriGotoh-Y** [reply]
如果用的是苹果手机那么小孩子是完全没有进绳网的希望了，毕竟想把IPA文件装进苹果手机还得签名信任再用电脑下进手机，隔段时间还得重新签，要不然就得买证书

**Reply by HitoriGotoh-Y** [reply]
基本很难火起来，毕竟我们是不合规版的贴吧（咳咳），首先作者不够知名也没多少人知道，再者知道了普通人也不会玩，在这里的大部分都是会编程玩互联网的学生或程序员，反正至少会挂个梯子会点英语才能进来

**Comment by levia114514van** [comment]
这个素质水平的真能用得了github吗:P

**Comment by fyhwsy** [comment]
没事，作者也清楚，大概率会搞答题库米哈游版，到时候可以屏蔽很多很多不聪明的人，而且绳网帖子超过5个举报就会被屏蔽，已经很严了<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >

**Reply by nostalgiatan** [reply]
不会搞题库，被破解了就等于白给。
这个后期再想办法，目前就这样。

**Comment by ddsgkky** [comment]
说实话，这个进来还是挺难搞的，xxs应该还是没那个耐心<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Destiny747** [comment]
注册账号就是一个门槛了

**Comment by homteam** [comment]
不会的因为能进来的xxs都能有一点技术的，素质不会太差

**Comment by CBBC123** [comment]
感觉不会吧，登进来都大费周章了(花了我两天时间)，应该不会有黑子愿意画那么大功夫进来闹事吧

**Reply by bangbu01** [reply]
？啊？二天？

**Comment by fa-e-tong-LKN** [comment]
我觉得最大的门槛应该是耐心吧，xxs们明明有更方便的也不会选择这里，能超越耐心而来的应该是有丁尼任务吧，但至少很长一段时间应该是不会有丁尼任务延伸到这里的<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Maple013** [comment]
让我们细数一下使用绳网的必备条件吧：
1.拥有一台可以独立支配的电脑，或懂得如何隐藏下载的加速器、油猴等必备组件；
2.拥有一个qq号并懂得网页版qq邮箱的基本使用，或拥有/可借到其他可用于注册github的电子邮箱；
3.拥有基本的英语阅读能力，或能熟练使用翻译软件、搜索引擎，以适应github全英的网站页面；
4.能独立下载WT等能用于访问github的加速器，或懂得使用电脑上已经安装的加速器；
5.能够独立注册一个github账号，或能借到/偷偷使用朋友/家人的github账号；
6.能够发布和回复绳网的帖子；条件降低点的话，知道回复时右下角绿色的“Comment”键能起到“回复”键的作用。
综上所述，如果一个低龄小孩能上来尝试爆破绳网，最有可能的情景是：
父母从事计算机相关工作，将个人用机拿给孩子玩（工作机不现实），无人监管的孩子在B站发现绳网，并藉由耳濡目染登录github、上父母账号、大量发帖试图爆破绳网但很快被清除影响。
至于次日，家长是工作时发现昨天出现了大量页面贡献还是发现有人尝试联系账号的拥有者，发觉事情不对并导致始作俑者得到惩罚，那都是后话了。

**Reply by Maple013** [reply]
至于其他有技术的成熟人类前来爆破，我的观点与上面几位一致：有需求或有能力注册了github账号的人，他所接受的教育不支持他做出这种撒泼打滚式的行为。
不过，万一，万一真有魔怔反米人冲了进来，我查了一下，所有者是拥有讨论的管理权限的，手动清理一下再上黑名单即可，大不了临时关闭讨论等几天再开。
那万一万一有人拉了一票github账号24*365无死角轰炸……
不是， **他没病吧？** 有这能耐不去B站抖音米游社引导舆论，闲着没事来这同人小网站闹腾？
# **他有病吧？**

**Reply by shxtxdy** [reply]
进来这个网站肯定是没有那么困难的，手机上登录github，我记得当时好像挂个加速器，用邮箱登一下就行了，有点太久了，忘了，而翻译语言障碍已经不成问题了，手机自带的浏览器自带的都能翻译，而手机上有的浏览器什么的也支持脚本下载，所以说只要他把账号注成了，然后再下个脚本就行了，手机上好多浏览器都是支持下载用户脚本的，不一定非要用电脑，所以综上来说，他要是有个加速器会翻外网，那他就能注册，因为在外网要用邮箱登录，邮箱肯定是有的，qq就能注册简单，注册这些东西都不需要什么教养，这又不难，这种人真进来的话都不会只有一两个，删肯定是删不过来的，而以后会出现题库什么的，他只是魔怔，不是傻，咱们都能进来，他肯定也能进，所以太火的话肯定是会被冲的

**Reply by nostalgiatan** [reply]
没事不慌，github做基础局面不会太差，反正规矩已经摆那了，敢带节奏就有人敢举报，现在绳网上的人要是不立规矩，啥都可以给你安个名头举报了，他们要是敢来发一个帖子，估计就要10分钟隐藏了，如果他们敢爆破评论区，也可能会10分钟隐藏了，后期让管理把那些评论删了之后再把帖子放出来问题也是不太大的。

**Reply by dnmdngcsb** [reply]
不是，其实10代以后小老登的动手能力是很弱的，很少有人会熟练操作电脑，就算会的素质也不会那么低，因为大家都用手机，不过我就是10年的，很早有VPN并且有github账号了

**Reply by nninng** [reply]
为什么我用那个加速器后感觉网速变差了，根本点不进来，我把加速器一关立马加载出来了

**Comment by Ziew-Ryan** [comment]
不太可能，使用门槛很高
1.可以自由支配电脑
2.懂得下载并使用浏览器脚本
3.有自己的电子邮箱
4.有翻越Firewall的手段或者运气爆棚可以直连
最后，其实语言也没多大阻碍，大部分浏览器都有翻译，最后的最后，小学生也不全是没素质的人，就比如我😁

**Comment by ZiGuiti4029** [comment]
感觉门槛还挺高的，xxs估计没那个耐心跟技术<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by cx525** [comment]
能正确安装steam都超过网上99%的人了

**Reply by dnmdngcsb** [reply]
是这样的，看看tb上一堆卖正版steam安装包的销量吧

**Comment by QingMingqwq** [comment]
进来的门槛挺高的，应该......不会吧

**Comment by XmLjsk** [comment]
先不说其他的，没点脑子还真的连GitHub账号都不会注册🤓

**Comment by iuhjk** [comment]
不知道，现在变化太快了。像我当时上高中的时候（也没多久，就5、6年吧）根本没手机，上大学才有手机，那时候哪儿知道这些东西。现在的学生几乎人手一个手机。

**Reply by AnonCN** [reply]
放心，github这个网站还是有一丢丢门槛的

**Comment by lixiao189** [comment]
不用担心，众所周知，只有拥有 4 大 AI 的传奇神匠才能攻破绳网，各位绳匠不用担心

**Reply by AnonCN** [reply]
入戏太深了兄弟（

**Comment by aizai1173** [comment]
b站答题如果改回以前，应该就不会有这么多神人了

**Comment by lky1433223** [comment]
还是有门槛的，至少需要会安装油猴和插件，还要有github账号，掌握简单的github使用技能。

**Comment by Aries-Woof** [comment]
**### _![IMG_4704](https://github.com/user-attachments/assets/0f9be275-f65b-4172-adf1-39f2cecdf834)_**

**Comment by Aries-Woof** [comment]
倒是好奇以后这网站会怎么发展？

**Comment by liu20040515** [comment]
没点耐心不会来的

**Comment by tanls930** [comment]
没事，我们这不是真有社管吗（doge

**Comment by LuAn-afl** [comment]
感觉已经开始了就有个人一直发图我一刷新绳网有一页都是他的看得我审美疲劳

**Comment by ybbgh** [comment]
可以防低龄的，但是没素质的难防<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Comment by yivitte** [comment]
低龄的都过不了注册账户这一关，不用担心，但是如果圈子大了，低素质的人就很难避免了，只希望到时候可以招募多一些人来负责监管。但是为爱发电总是有个尽头的，所以我并不看好

**Comment by 15666851105** [comment]
讲真的，头一次觉得学了10多年英语这么有用😐

**Reply by 1ctrl-cv** [reply]
然而油叉上有github中文化插件（

**Comment by 1ctrl-cv** [comment]
光上github就能拦住很大一部分

**Comment by Fukacis** [comment]
不会吧<img src="https://img-static.mihoyo.com/communityweb/upload/76cc94063d9c0efb2335e8487299cd0a.png" alt="银狼 吹泡泡" width="75" >光是登录就有点麻烦了，真的有人闲到来这儿刷存在感吗<img src="https://bbs-static.miyoushe.com/static/2024/07/31/ed2b65b6899fc16528b1d10adfcb9a38_8705924363701631267.png" alt="云璃 白眼" width="75" >感觉难说

**Comment by vioWistaria** [comment]
想兴盛起来都不容易，说真的，只要别淹没在网络上就不错了。

**Comment by fa-e-tong-LKN** [comment]
考古一下，大家咋样

**Comment by 2213653** [comment]
我是xxs… 其实素质没多差

**Comment by 1talan** [comment]
感觉现在有点危险，站长搞了软件版，登陆创建更简单了，我没看教程30分钟搞定了，我手笨，本来想在b站找大佬让别人帮忙的

**Reply by zei-21** [reply]
有软件版？？？

**Comment by IsaacNery1** [comment]
不到啊，我手机版搞了半小时才注册下来，主要是梯子太卡了人机验证断了几次，别个应该没有这个耐心搞事叭

**Reply by zei-21** [reply]
有，比如我

**Comment by hsd221** [comment]
不会的喵，注册github也没有那么容易的喵

**Reply by zei-21** [reply]
还行，也就我电脑用虚幻5进行实机渲染所需时间，but 容易断连

**Comment by xphorror** [comment]
估计难，不过看底下很多人在说光账号能拦住大部分人，我觉得并不绝对。

以我自己为例，我现在14岁，21年接触外网，自己注册的github和steam，有自己的梯子

可能是兴趣使然吧，目前能够编写简单的Batch脚本

**Reply by skycat2216** [reply]
但這種程度也不是小鬼了，小鬼通常沒這麼有能
我甚至沒有自己的梯子(Ex紳士、DMM也會用到)，我還不是在這邊？

**Comment by 1talan** [comment]
手机端已经打不开网址了。发生什么了



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年12月29日 23:32 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] 话说以后这个项目大了之后会不会有一些低龄低素质的人啊？林子大了什么人都有，b站的社区环境已经搞得乌烟瘴气的了，有点怕这里也会沦陷 (Discussion #2325) |

估计难，不过看底下很多人在说光账号能拦住大部分人，我觉得并不绝对。

以我自己为例，我现在14岁，21年接触外网，自己注册的github和steam，有自己的梯子

可能是兴趣使然吧，目前能够编写简单的Batch脚本

**Comment by jane-10086-jpg** [comment]
应该不会，毕竟墙内墙外还要翻。

**Comment by LAIN9527** [comment]
我注册了git账号但是绳网应用里面还是没法登录谁有办法吗

**Reply by LAIN9527** [reply]
我是手机端的

**Comment by HitoriGotoh-Y** [comment]
这倒不至于，进绳网还是有点要求的，最起码进Github已经难住很多人了，没个加速器或者梯子进Github都很麻烦，更别说进绳网了，还有绳网电脑端软件和绳网iOS端软件更是难上加难

**Comment by zei-21** [comment]
不一定











在 2025-11-30 13:18:43，"後藤ひとり" ***@***.***> 写道：

基本很难火起来，毕竟我们是不合规版的贴吧（咳咳），首先作者不够知名也没多少人知道，再者知道了普通人也不会玩，在这里的大部分都是会编程玩互联网的学生或程序员，反正至少会挂个梯子会点英语才能进来

**Reply by youyiMC** [reply]
俺寻思绳网也不是什么合法网站（doge）

**Comment by youyiMC** [comment]
小鬼想登上Github都有难度罢，之前反米吧还有想要DDoS攻击这里的（爆笑）

**Reply by skycat2216** [reply]
笑死

**Reply by zhengyang3552** [reply]
绷

**Reply by wxt-170** [reply]
门槛高了傻子自然进不来了😋

**Comment by qingyangprince** [comment]
不会再大了，这个项目已经有点死了😭

**Comment by AAAEllenJoe** [comment]
林子里的傻鸟会操作的应该挺少的吧
